client/src/components/home/index.tsx

```diff
@@ -82,8 +82,14 @@ function HomePage(props: {
     selectSubject,
     saveChanges,
     startTraining,
+    launchMentor,
   } = useWithReviewAnswerState(props.accessToken, props.search);
 
+  const buildAction = () =>
```

Comment:
is there a case for performing any build/launch action if the mentor is undefined?

Maybe this should be

```typescript
const continueAction = () => {
  if(!mentor) {
    console.error("attempt to build or preview a mentor but no mentor set")
    return
  }
  if (mentor.isDirty) {
    startTraining(mentor._id)
  }
  else {
    launchMentor(mentor._id);
  }
}